users/src/main/java/io/crate/operation/auth/AuthenticationService.java

```diff
@@ -38,9 +43,27 @@
     private static final String KEY_METHOD = "method";
 
     private final Map<String, AuthenticationMethod> authMethodRegistry = new HashMap<>();
+
+    @VisibleForTesting
+    protected Map<String, Map<String, String>> hbaConf() {
+        return hbaConf;
+    }
+
     private Map<String, Map<String, String>> hbaConf;
```

Comment:
@mfussenegger the structure of hbaConf is explained here https://github.com/crate/crate/blob/3ed7e26e4d5d5c000827c8974160c952e2ffa806/users/src/main/java/io/crate/operation/auth/AuthenticationService.java#L70